content/articles/2017-11-01-lubridate-1.7.0.Rmd

```diff
@@ -0,0 +1,135 @@
+---
+title: lubridate 1.7.0
+slug: lubridate-1.7.0
+description: >
+  A new version of lubridate is on CRAN!
+author: Vitalie Spinu
+date: 2017-11-01
+photo:
+  url: https://unsplash.com/photos/yBzrPGLjMQw
+  author: Heather Zabriskie
+categories: [package]
+---
+
+```{r setup, include = FALSE}
+library("lubridate")
+knitr::opts_chunk$set(collapse = TRUE, comment = "##=>")
+```
+
+We are pleased to announce that `lubridate` 1.7.0 is now on CRAN! For a complete
+set of changes please see the release
+[NEWS](https://github.com/tidyverse/lubridate/releases/tag/v1.7.0). Here is a
+brief walk-through of the prominent new features.
+
+## Built-in CCTZ and much faster `update` and `force_tz`
```

Comment:
Does this mean lubridate now has a C++11 dependency?

Reply:
Yes. Rcpp dependency with C++11 standard. Are there any issues with that?

Reply:
There's a minor issue because some risk averse organisations use RHEL, and one of the common stable versions has a very old version of gcc. I think we can just wait to see if this is a problem.

Reply:
>  some risk averse organisations use RHEL

And here we [go](https://github.com/tidyverse/lubridate/issues/601).